packages/server/src/middlewares/logError.ts

```diff
@@ -9,7 +9,12 @@ export { logError };
 
 const isProd = process.env.NODE_ENV === "production";
 
-const logError = (err: Error, _: Request, res: Response) => {
+const logError = (
+  err: Error,
+  _: Request,
+  res: Response,
+  next: NextFunction
```

Comment:
@remiriv pk on rajoute cet argument `next`?

Reply:
https://stackoverflow.com/questions/65627437/typeerror-res-status-is-not-a-function-in-a-error-handling-middleware sinon apparemment il comprend pas `res.status` parce qu'il considère que le dernier argument est next et pas res

Reply:
@remiriv ah oui j'avais pas compris que c'était un middleware. Good!